posthog/api/property_definition.py

```diff
@@ -273,6 +282,48 @@ def _join_on_event_property(self):
             else ""
         )
 
+    @staticmethod
+    def __property_name_aliases():
+        """
+        Generates a SQL CASE statement that aliases property names to their label.
+        """
+        return (
+            "CASE name "
+            + " ".join(f"WHEN '{k}' THEN '{v}'" for (k, v) in PROPERTY_NAME_ALIASES.items())
+            + " ELSE name END as __name_alias"
```

Comment:
Maybe else `NULL`, since `search_fields` already includes `name`, and it may be a bit slower to effectively search through `name` twice

Reply:
Query changed, doesn't do this any more